gravity.sh

```diff
@@ -17,6 +17,8 @@ coltable="/opt/pihole/COL_TABLE"
 source "${coltable}"
 regexconverter="/opt/pihole/wildcard_regex_converter.sh"
 source "${regexconverter}"
+readonly databaseMigrationScript="/etc/.pihole/advanced/Scripts/database_migration/gravity-db.sh"
+source "${databaseMigrationScript}"
```

Comment:
This variable can be inlined

Reply:
I did this for consistency with the other lines, we can surely inline it.

Reply:
As long as it isn't used anywhere and inlining doesn't cause any other issues, go ahead with it.

Reply:
This produces a lint error now. I guess that's why it's done like this for the other lines as well.

> Not following: /etc/.pihole/advanced/Scripts/database_migration/gravity-db.sh was not specified as input (see shellcheck -x).

Reply:
I added the corresponding linting instruction that it is okay to not follow this file. This solves the problem. As the exception needs to be added individually for each `source` call, this does not interfere with the checking of the remaining source code in this file.